packages/amplication-client/src/Workspaces/ResourceList.tsx

```diff
@@ -45,6 +45,12 @@ function ResourceList() {
     setError(null);
   }, [setError]);
 
+  const handleResourceClick = () => {
+    trackEvent({
+      eventName: AnalyticsEventNames.UpgradeOnResourceListClick,
```

Comment:
According to the Notion docs you need to add this metadata:
Reason: “Entities limit notification”
Location: App

Reply:
I can't add it. we need to move stigg provider to higher level in order to get the reason for the notification

Reply:
move stigg provider #4856